### DISCUSSION TITLE
How can one person install this one on our machines?

### DISCUSSION THREAD

**Discussion by JTech2022** [input-leap/input-leap#1605] [Q&A] [answered]
### What happened?

Till now i cant install this on a machine

Any solution till now?

### Version

v2.4.0

### If applicable, where did you install Input Leap from?

Not yet

### What OSes are you seeing the problem on? (Check all that apply)

Linux, Windows

### What OS versions are you using?

Windows and MAC

### Any other information

Any update will be awesome

**Comment by shymega** [comment, marked as answer]
This is an inappropriate issue, so I've converted it into a discussion. Issues are for bugs or regressions.

No, there are no proper releases yet.

**Reply by Rings4** [reply]
How do we run this then?

**Reply by shymega** [reply]
@Rings4 You can't yet. There are builds for Windows on CI, not macOS yet, but it's still being refactored.

**Reply by Rings4** [reply]
I see. I'm on Windows. How would I run this?

**Reply by jw834ksj43** [reply]
Is there currently an expected first release date or time frame?

**Reply by shymega** [reply]
> Is there currently an expected first release date or time frame?

@jw834ksj43 No.



> I see. I'm on Windows. How would I run this?

There's no installer yet, we're waiting for a CI PR to be approved. Once that's in mainline, installer builds will be produced for Windows (but unsigned).

**Reply by Rings4** [reply]
Good to know, thanks! Will an update be posted here when it's out, or should I follow another topic to be informed?

**Reply by shymega** [reply]
@Rings4 Sorry for the delay. Probably not, I'm involved in a fair amount of projects, and barely have time to notify in individual threads.

What might be best is that you wait for v3.0.0, and you can get alerts on GitHub by 'watching' for releases. I trust this page from StackOverflow may help. https://stackoverflow.com/a/49350626

**Comment by sithlord48** [comment]
https://github.com/input-leap/input-leap/discussions/1793#discussioncomment-8577620
